src/tests/topp/ConvertTSVToTraML_2_output.TraML

```diff
@@ -81,6 +81,11 @@
       <cvParam cvRef="MS" accession="MS:1001226" name="product ion intensity" value="1"/>
       <cvParam cvRef="MS" accession="MS:1002007" name="target SRM transition"/>
       <userParam name="annotation" type="xsd:string" value="y5"/>
+      <userParam name="detecting_transition" type="xsd:integer" value="1"/>
```

Comment:
No, this is currently not done. So you could set some fields (e.g. detecting_transition) and for the others (e.g. identifying_transition), the default will be used.